src/ColumnsSummary.php

```diff
@@ -41,6 +41,11 @@ class ColumnsSummary
 	 */
 	protected $renderer;
 
+	/**
+	 * @var bool
+	 */
+	protected $positionTop;
```

Comment:
Is there any default value of this property? :)

```php
protected $positionTop = false;
```

Reply:
<img width="575" alt="image" src="https://user-images.githubusercontent.com/19396906/42464662-0b5bb2b0-83aa-11e8-81e0-9594a3501615.png">

false = false
(other) = true

true:
<img width="395" alt="image" src="https://user-images.githubusercontent.com/19396906/42464863-a26adc3a-83aa-11e8-87b0-c4fa5a4b399c.png">

false:
<img width="389" alt="image" src="https://user-images.githubusercontent.com/19396906/42464885-af66f586-83aa-11e8-9e95-f38dedbcfd4d.png">

Reply:
There is no initial call of `::setPositionTop()` so the default value is `null`. Please set the default value to false as I mentioned in the first comment.